Manifest.toml

```diff
@@ -9,23 +9,23 @@ version = "0.0.4"
 [[Base64]]
 uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 
+[[CodeTracking]]
```

Comment:
@pszufe - why do we need Manifest.toml in this package?

Reply:
@bkamins Why do you ask me? I am not the author of this PR. `Manifest.toml` should not be included.

Reply:
@pszufe - I ask you because Manifest.toml was already in the project - it was not added in this PR.

Reply:
@bkamins it is not (and should not be) present in https://github.com/pszufe/SimpleHypergraphs.jl/tree/master  
This is an issue of unresolved conflicts.

Reply:
@pszufe - ah, then sorry for mentioning you.

@aleant93 what I would recommend you is do the following.

1. Regarding this PR:
    - please rebase your branch against master
    - squash all commistt on top of master
    - force push the branch
(in this way it will be clear what is changed against master)

2. As a general workflow
    - I would recommend you to use `dev` in package manager when working on the registered package
    - I would recommend you to never activate the project environment associated with the package (as it is very easy to mess up the Project.toml then due to automatic updates)

I hope this helps. Thank you for working on this!

Reply:
@bkamins many thanks for the useful suggestions - I will work with this workflow from now on.